qutebrowser/browser/commands.py

```diff
@@ -1960,6 +1960,13 @@ def jseval_cb(out):
                         out = out[:5000] + ' [...trimmed...]'
                     message.info(out)
 
+        if file:
+            try:
+                with open(js_code) as f:
+                    js_code = f.read()
+            except FileNotFoundError:
```

Comment:
Please handle `OSError` here instead to handle other issues while reading the file as well, and add the exception message into the error message (do `as e:` here and I'd say just do `message.error(str(e))` or so below)